packages/typescript-estree/src/node-utils.ts

```diff
@@ -527,6 +527,7 @@ export function getTokenType(
 
   switch (token.kind) {
     case SyntaxKind.NumericLiteral:
+    case SyntaxKind.BigIntLiteral:
```

Comment:
Since we are here, could you also take a look at https://github.com/typescript-eslint/typescript-eslint/issues/11022? Many thanks.

Reply:
https://github.com/typescript-eslint/typescript-eslint/pull/11023 🎉